include/alpaka/atomic/AtomicStlLock.hpp

```diff
@@ -31,6 +31,9 @@ namespace alpaka
     {
         //#############################################################################
         //! The CPU threads accelerator atomic ops.
+        //
+        //  Atomics can used in the hierarchy level grids, blocks and threads.
+        //  Atomics are not guaranteed to be save between devices.
         //#############################################################################
         class AtomicStlLock
```

Comment:
The friend declaration at line 46 needs to be extended by `typename THierarchy`.